lib/web/JsonpChunkLoadingRuntimeModule.js

```diff
@@ -98,6 +99,12 @@ class JsonpChunkLoadingRuntimeModule extends RuntimeModule {
 			chunkGraph.getChunkConditionMap(chunk, chunkHasJs)
 		);
 		return Template.asString([
+			withBaseURI
+				? Template.asString([
+						`${RuntimeGlobals.baseURI} = document.baseURI || self.location.href;`
```

Comment:
technically this also should work
```suggestion
						`${RuntimeGlobals.baseURI} = document.baseURI || self.location;`
```

Reply:
I thought about making in consistent always a string...